### PR TITLE
Feature/us8196 leader resource redirect config

### PR DESCRIPTION
**Pull request by Doug-Shannon**
This also contains the upgrade to crossroads.web.common/apiKeys

### CODE REVIEW

File: crossroads.net/app/group_tool/services/groupUrl.service.js [hunk 1]
```diff
@@ -1,12 +0,0 @@
-export default class GroupURLService {
```
Comment (silbermm) [inline crossroads.net/app/group_tool/services/groupUrl.service.js]: did you intentionally delete this file?

Comment (Doug-Shannon) [inline crossroads.net/app/group_tool/services/groupUrl.service.js]: Yeah, it wasn't being used anywhere, I couldn't find any references to the file.

File: Gateway/crds-angular/MinistryPlatform.config [hunk 2]
```diff
@@ -214,6 +214,9 @@
   <add key="GroupLeaderConfirmationTemplate" value="2015"/>
   <add key="GroupLeaderReferenceEmailTemplate" value="2018"/>
   <add key="GroupLeaderNoReferenceEmailTemplate" value="2020"/>
+  <add key="GroupLeaderAppCode" value="GroupLeader"/>
```
Comment (silbermm) [inline Gateway/crds-angular/MinistryPlatform.config]: can we make this all Uppercase like all of the other configuration AppCodes?

File: CI/SQL/20170605_160500_US8196-add-group-leader-url-config.sql [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+GO
+
+-- =============================================
+-- Author:      Doug Shannon
+-- Create date: 2017-06-05
+-- Description:	Adds Group Leader URL Config Value
+-- =============================================
+
+DECLARE @Configuration_Setting_ID int = 301
+DECLARE @Application_Code VARCHAR(max) = 'GroupLeader'
```
Comment (silbermm) [inline CI/SQL/20170605_160500_US8196-add-group-leader-url-config.sql]: can we upcase this like all of the other AppCodes?

**Pull request merged**